DeskKit/DeskKit/View Controllers/DKContactUsViewController.m

```diff
@@ -134,12 +132,23 @@ - (void)setupViewModel
     
     self.viewModel.userIdentity = self.userIdentity;
     self.viewModel.subject = self.subject;
-    self.viewModel.toEmailAddress = self.toEmailAddress;
     
     self.viewModel.includeYourNameItem = self.showYourNameItem;
     self.viewModel.includeYourEmailItem = self.showYourEmailItem;
     self.viewModel.includeSubjectItem = self.showSubjectItem;
     self.viewModel.customFields = self.customFields;
+    
+    if (self.toEmailAddress == nil) {
+        __weak typeof(self) weakSelf = self;
+        [[DKSession sharedInstance] hasContactUsToEmailAddressWithCompletionHandler:^(BOOL hasContactUsToEmailAddress) {
+            __weak typeof(self) strongSelf = weakSelf;
```

Comment:
I think we should remove `__weak`, because as I understand your intention is to hold a strong reference until this block is complete.